File: Chapter 02/Python programs/2_15.py
a=int(input("Enter marks in subject 1: "))
b=int(input("Enter marks in subject 2: "))
c=int(input("Enter marks in subject 3: "))
d=int(input("Enter marks in subject 4: "))
e=int(input("Enter marks in subject 5: "))

p=((a+b+c+d+e)/500)*100
if  p >=90:
    print("Grade is A")
elif  p >=80:
    print("Grade is B")
elif  p >=60:
    print("Grade is C")
elif  p >=40:
    print("Grade is D")
else:
    print("Grade is F")

File: Chapter 02/C Programs/2_15.c
#include<stdio.h>
void  main()
{
int a,b,c,d,e;
float p;
printf("Enter marks in  5 subjects ");
scanf("%d%d%d%d%d", &a,&b,&c,&d,&e);
p=((a+b+c+d+e)/(float)500)*100;
if (p >=90)
printf("Grade is A");
else  if  (p >=80)
printf("Grade is B");
else if ( p >=60)
printf("Grade is C");
else if  (p >=40)
printf("Grade is D");
else
printf("Grade is F");
}
